### ISSUE TITLE
Banned styles -moz-binding and behavior pass on validation

### ISSUE THREAD

**Issue by alabiaga** [ampproject/amphtml#13701]
According to the docs related to styles, -moz-binding and behavior are banned styles, though I can use them and they are not flagged as so.

## How do we reproduce the issue?

https://validator.ampproject.org/

1. Add the following style in the validator web ui

 <style amp-custom>
   .test {
     -moz-binding: inherited;
     behavior: url(http://www.msn.com/blah.htc);
    }
  </style>

**Comment by alabiaga** [comment]
/cc powdercloud

If this is valid, please assign to me and I can fix. Thanks

**Comment by honeybadgerdontcare** [comment]
Yes, this is valid now per #12352 conversation with Dima. Documentation needs to be updated.

**Comment by alabiaga** [comment]
Great. Thanks
